pkgs/applications/graphics/krop/default.nix

```diff
@@ -19,6 +19,11 @@ python3Packages.buildPythonApplication rec {
     ghostscript
   ];
 
+  nativeBuildInputs = [ qt5.wrapQtAppsHook ];
+  makeWrapperArgs = [
```

Comment:
It should not be necessary to set `makeWrapperArgs`, particularly because this should be the default value.

Reply:
Hm, could it be that the `buildPythonApplication` interferes?

```
# When I remove the makeWrapperArgs
$ git diff
diff --git a/pkgs/applications/graphics/krop/default.nix b/pkgs/applications/graphics/krop/default.nix
index 401e5f6fc57..1455323b45e 100644
--- a/pkgs/applications/graphics/krop/default.nix
+++ b/pkgs/applications/graphics/krop/default.nix
@@ -20,9 +20,6 @@ python3Packages.buildPythonApplication rec {
   ];
 
   nativeBuildInputs = [ qt5.wrapQtAppsHook ];
-  makeWrapperArgs = [
-   "\${qtWrapperArgs[@]}"
-  ];
 
   # Disable checks because of interference with older Qt versions // xcb
   doCheck = false;

# And try to run the newly build krops, it fails with the same error as the linked issue.

[maarten@maarten-yoga:~/code/nix/nixpkgs/krop]$ $(nix-build  -A krop)/bin/krop
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted (core dumped)

# Reverting this state

[maarten@maarten-yoga:~/code/nix/nixpkgs/krop]$ git stash -- pkgs/applications/graphics/krop/default.nix
Saved working directory and index state WIP on krop-wrapQtAppsHook: 7d14a45b515 krop: Add wrapQtAppsHook. Closes #77339

[maarten@maarten-yoga:~/code/nix/nixpkgs/krop]$ $(nix-build  -A krop)/bin/krop
^C

# makes the program run properly 

[maarten@maarten-yoga:~/code/nix/nixpkgs/krop]$ 
```

Reply:
Ah! That must be it.